plugins/npm/src/index.ts

```diff
@@ -1004,9 +1011,7 @@ export default class NPMPlugin implements IPlugin {
             "--yes",
             "--no-push",
             "-m",
-            isIndependent
-              ? `'"Bump independent versions [skip ci]"'`
-              : `'"Bump version to: %s [skip ci]"'`,
+            `'"${commitMessage}"'`,
```

Comment:
We should ensure that [skip-ci] is on this commit even if it's customize. maybe a check to see if it's already in the string?

Reply:
Done and added a test.